modules/activitypub/user_settings.go

```diff
@@ -5,16 +5,19 @@ package activitypub
 
 import (
 	user_model "code.gitea.io/gitea/models/user"
+	"code.gitea.io/gitea/modules/util"
 )
 
+const rsaBits = 2048
```

Comment:
This is rather low these days - does it have to be 2048?

Reply:
https://github.com/chef/chef/blob/188364ff4da41dde97b0d9639bf55f0ac35158d9/lib/chef/api_client/registration.rb#L183-L185

But that was added 2014. I will change it to 4096; there was no problem in my tests.

Reply:
I see that's not my change but was from the acitivity pub code. Can't tell why it is 2048 there. The Chef code now uses 4096 bits.